_includes/project-teaser-slim.html

```diff
@@ -27,18 +27,21 @@ <h4 class="project-title"><a href="{{ sitepage.url }}"> {{ sitepage.title }}</a>
       </div>
 
       <div class="project-footer clearfix">
-
-        {% for lab in site.labs %}
-          {% if lab.lab == sitepage.lab %}
-            {% assign laburl = lab.key %}
-            {% break %}
-          {% endif %}
+        {% for project_lab in sitepage.lab %}
+          {% capture is_last_one_in_loop %}{% if forloop.last %}true{% endif %}{% endcapture %}
```

Comment:
ja, der hat es leider nicht in `3.0.6` von `liquid` geschafft. 
Ein einfaches assign sollte aber abhilfe verschaffe:
```
 {% assign outer_forloop = forloop %}
```